src/display/editor/stamp.js

```diff
@@ -66,7 +68,11 @@ class StampEditor extends AnnotationEditor {
             this.remove();
             return;
           }
-          ({ bitmap: this.#bitmap, id: this.#bitmapId } = data);
+          ({
+            bitmap: this.#bitmap,
+            id: this.#bitmapId,
+            svg: this.#isSvg,
```

Comment:
Shouldn't this be as follows, looking at the code below in this file?
```suggestion
            isSvg: this.#isSvg,
```